src/helpers.js

```diff
@@ -87,18 +93,34 @@ export function applySugar(environment: Object) {
 }
 
 export function getCommand(): { command: string, options: Object, parameters: Array<string> } {
+  const pythonFile = Path.join(Path.dirname(__dirname), 'index.py')
```

Comment:
Just do `__dirname, '..'`, no sense in doing this silly "dirname of a dirname" code.